src/Catel.Test/Catel.Test.Shared/IoC/TypeFactoryFacts.cs

```diff
@@ -370,6 +372,49 @@ public void CreatesTypeWithPropertyInjection()
 
                 Assert.IsNotNull(instance.UiVisualizerService);
             }
+
+            [TestCase]
+            public void IfTypeFactoryIsCalledConcurrentlyItRunsFasterThatSerial()
```

Comment:
Serial vs Parallel like physical ports. maybe sequential is better word. I will try to fix all issues soon. Now I'm on vacation with limited access to computer.